Web/starter_project_iconics/pages/teams/index.tsx

```diff
@@ -1,8 +1,9 @@
-import TeamsHero from '@/components/team/TeamsHero'
+import Teams from '@/components/team/Teams'
+
 
 function TeamsPage() {
   return (
-    <TeamsHero></TeamsHero>
+    <Teams></Teams>
```

Comment:
If you're not going to add children you dont need closing tags, you can just use ```<Teams />```